src/styles/helpers/parseBorderWidth.ts

```diff
@@ -0,0 +1,15 @@
+import { OakBorderWidthToken, oakBorderWidthTokens } from "../theme";
+
+import pxToRem from "./pxToRem";
+
+export function parseBorderWidth(
+  value: OakBorderWidthToken | null | undefined,
+) {
+  if (!value) {
+    return undefined;
+  }
+
+  if (value in oakBorderWidthTokens) {
+    return `${pxToRem(oakBorderWidthTokens[value])}rem`;
+  }
+}
```

Comment:
`parseBorder` also applies a `border-style` E.g `0.125rem solid` which won't work with `stroke-width`. So i've added `parseBorderWidth` which follows the same conventions but only returns the size.